service/src/main/java/uk/gov/hmcts/reform/fpl/service/ChildrenService.java

```diff
@@ -80,6 +81,48 @@ public boolean allChildrenHaveFinalOrder(List<Element<Child>> children) {
         return children;
     }
 
+    public List<Element<Child>> updateFinalOrderIssued(CaseData caseData) {
+        return updateFinalOrderIssued(
+            caseData.getManageOrdersEventData(),
+            caseData.getAllChildren(),
+            caseData.getOrderAppliesToAllChildren(),
+            caseData.getChildSelector(),
+            caseData.getRemainingChildIndex()
+        );
+    }
+
+
+    private List<Element<Child>> updateFinalOrderIssued(ManageOrdersEventData manageOrdersEventData,
```

Comment:
We already have a method with the same name. Would it be worth reusing the code, rather than duplicating it?